### PR TITLE
Rest Neutrals & Code Improvements

### PR DESCRIPTION
**Pull request by Ultradragon005**
- [x] Neutral Benign
- [x] Neutral Chaos
- [x] Automate CreateRoleClass

### CODE REVIEW

**Comment by Ultradragon005** [comment]
i forgor, change this to RoleBase

File: Patches/PlayerControlPatch.cs [hunk 1]
```diff
@@ -1161,245 +1161,6 @@ public static bool Prefix(PlayerControl __instance, [HarmonyArgument(0)] GameDat
...
-                if (__instance.Is(CustomRoles.Unlucky) && (target?.Object == null || !target.Object.Is(CustomRoles.Bait)))
-                {
-                    var Ue = IRandom.Instance;
-                    if (Ue.Next(1, 100) <= Unlucky.UnluckyReportSuicideChance.GetInt())
-                    {
-                        Main.PlayerStates[__instance.PlayerId].deathReason = PlayerState.DeathReason.Suicide;
-                        __instance.RpcMurderPlayerV3(__instance);
-                        return false;
-                    }
-                }
```
Comment (Tommy-XL) [inline Patches/PlayerControlPatch.cs:1402]: Bruh

Comment (Ultradragon005) [reply Patches/PlayerControlPatch.cs:1402]: oops

**Comment by Ultradragon005** [comment]
image
crazyyy as conflict  ngl💀

**Comment by Ultradragon005** [comment]
I realized I could just do `bool ?  Attribute.Attribute :  null`, so prolly gonna do #758 again, after done wid roles.

**Pull request merged**